docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1354,35 +1354,27 @@ public class RecordsResource {
         // ...
     }
 
+    @GET
+    @Path("/search/{text}")
+    @RestLink(rel = "search records by free text")
+    @InjectRestLinks
+    public List<Record> search(@PathParam("text") String text) { <4>
+        // ...
+    }
+
     @DELETE
     @Path("/{id}")
     @RestLink
-    public TestRecord delete(@PathParam("id") int id) {
+    public TestRecord delete(@PathParam("id") int id) { <5>
         // ...
-    }
+    }    
 }
 ----
-
-When calling the endpoint `/records` which is defined by the method `getAll` within the above resource using curl, you would get the web links header:
```

Comment:
I don't think we should remove these sections, they're useful because they provide a global view of the results we generate, while the notes you added make them scattered. I mean, the notes and explanations are useful, but so is the global result. BTW, I myself would have included the JSON links as well, because I find link headers less useful.

Reply:
ok ok, I will revert it back and add a new section for these changes.

Reply:
updated